src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -90,8 +90,8 @@ export class StudentExamResolve implements Resolve<StudentExam> {
      * @param route Contains the information about the route to be resolved
      */
     resolve(route: ActivatedRouteSnapshot): Observable<StudentExam> {
-        const courseId = route.params['courseId'] ? route.params['courseId'] : null;
-        const examId = route.params['examId'] ? route.params['examId'] : null;
+        const courseId = route.params['courseId'] ? route.params['courseId'] : undefined;
+        const examId = route.params['examId'] ? route.params['examId'] : undefined;
```

Comment:
This can be simplified.
```suggestion
        const courseId = route.params['courseId'] || undefined;
        const examId = route.params['examId'] || undefined;
```